bin/configtest

```diff
@@ -43,16 +43,19 @@ var reset   = '\u001b[0m';
     }
 
     var jpegOptimizer = config.jpegOptimizer;
-    stdout.write(magenta + "Global Optimizer:       " + reset + (jpegOptimizer || "Not set") + "\r\n");
+    stdout.write(magenta + "Global Optimizer:        " + reset + (jpegOptimizer || "Not set") + "\r\n");
 
     var bucket = config.bucket;
-    stdout.write(magenta + "Destination bucket:     " + reset + (bucket || "[Same bucket]") + "\r\n");
+    stdout.write(magenta + "Destination bucket:      " + reset + (bucket || "[Same bucket]") + "\r\n");
 
     var acl = config.acl;
-    stdout.write(magenta + "Global S3 ACL:          " + reset + (acl || "Not set") + "\r\n");
+    stdout.write(magenta + "Global S3 ACL:           " + reset + (acl || "Not set") + "\r\n");
 
     var cacheControl = config.cacheControl;
-    stdout.write(magenta + "Global S3 CacheControl: " + reset + (cacheControl || "Not set") + "\r\n");
+    stdout.write(magenta + "Global S3 CacheControl:  " + reset + (cacheControl || "Not set") + "\r\n");
+
+    var keepOriginalExtension = config.keepOriginalExtension;
```

Comment:
I think `keepExtension` is slightly simpler than `keepOriginalExtension`. Any thoughts?

Reply:
Right... I'll change name as you pointed.